keras/layers/recurrent.py

```diff
@@ -123,6 +161,11 @@ class Recurrent(Layer):
             enabling more time-efficient parallelization on the GPU.
             Note: RNN dropout must be shared for all gates,
             resulting in a slightly reduced regularization.
+            If set to 3 (LSTM/GRU only),
+            not only the RNN combines the gates into a single matrix,
+            but it also uses large matrices for input
+            Note: RNN dropout must be shared for all gates,
+            resulting in a slightly reduced regularization.
```

Comment:
no, I think there is no reference to this and it's not a big deal actually
it "cloud be" beneficial on some hardwares, which are good at multiplying big and medium matrices